nflow-engine/src/main/java/io/nflow/engine/workflow/definition/StateExecution.java

```diff
@@ -36,6 +37,14 @@
   int getRetries();
 
   /**
+   * Return the time when this workflow instance was originally scheduled to run.
```

Comment:
```suggestion
   * Return the time when this workflow instance was previously scheduled to run.
```

Reply:
It is the requested start time of the currently running step, not any previous step

Reply:
Then "Return requested start time of the currently running step." And actually that time usually is set in the previous step.